### PR TITLE
[STRAT-154] update packages

### PR DESCRIPTION
**Pull request by danielq987**
Updated to latest packages which are compatible with python 3.6-3.9.

### CODE REVIEW

**Review by ufshk — APPROVED** [review]
Seems fine. Approving. @e-wai any additional comments?

File: .gitignore [hunk 1]
```diff
@@ -12,6 +12,7 @@
 ehthumbs.db
 Thumbs.db
 .idea/*
+.python-version
```
Comment (e-wai) [inline .gitignore:15]: What does this ignore?

**Review by e-wai — APPROVED** [review]
Approving, but I will try to run on a separate branch

File: .gitignore (hunk 1 above)
Comment (danielq987) [reply .gitignore:15]: I used a python version manager to handle different python versions and virtual environments, and it generates a .python-version file

**Pull request merged**